CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG for 2.x
 =================
 
+* 2.4.2 (xxxx-xx-xx)
+
+    * Revert automatic cancellation of pending collection promises once the
+      output promise resolves. This was introduced in 42d86b7 and was both
```

Comment:
This has bee 1 commit from the #36 PR, so i decided to use the commit. Will add the PR and version 👍